README.md

```diff
@@ -32,6 +32,8 @@ client = Scnnr::Client.new do |config|
   config.timeout = 0 # sec
   config.logger = Logger.new(STDOUT) # You can specify an alternative logger.
   config.logger.level = :info # You can change the default log level.
+  config.retry_limit = 5 # How many retries of the request should happen
```

Comment:
@danimal141 
So your suggestion is it shouldn't be configurable - it's enough to just hardcode `retry_limit` and `sleep_on_retry`? What could be the values? 3 for `retry_limit` and 1 for `sleep_on_retry`?

Reply:
I've decided to act on your opinion which results in [this commit](https://github.com/NEWROPE/scnnr-ruby/pull/23/commits/d28daba717fb39b3fa6cfc205bccd0d05a17eb07). 
Let me know if that's what you had in mind!

Reply:
> it's enough to just hardcode retry_limit and sleep_on_retry? 

Yes, in my opinion, it's enough to decide the values internally.

> What could be the values? 3 for retry_limit and 1 for sleep_on_retry?

Looks good!

About `timeout`, I guess users should control the setting because they may not wanna wait for a response for a long time.
But about retrying, I guess all users want is just getting a response and they are not interested in how many times the client retries 🙂

Reply:
Gotcha :slightly_smiling_face: Thanks for the explanation!